src/java.base/share/classes/java/lang/reflect/code/bytecode/LocalsTypeMapper.java

```diff
@@ -233,6 +323,11 @@ private void accept(int elIndex, CodeElement el) {
                         pop(2);
                 }
             }
+            case IncrementInstruction i -> {
+                Slot v = locals.get(i.slot());
+                v.writes++;
```

Comment:
Why is this required? I likely don't understand the significance of this and how it differs from a store instruction.

Reply:
I've just pushed a different approach - load and store + link two var fragments.